components/cluster-secret-store-rh/production/kflux-rhel-p01/rhel-appsre-vault-approle.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: rhel-appsre-vault-approle
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
Since this seems to be a common annotation to all resources, should this get applied as an annotation in the `kustomization.yaml` manifest?

Reply:
Done